### PR TITLE
Update Toad.gd to allow for easier editing

### PR DESCRIPTION
**Pull request by Kuma-Boo**
# Description of changes

Enables a live preview of toad's color palette in the editor.

### CODE REVIEW

File: classes/interactable/npc/toad/toad.gd [hunk 1]
```diff
@@ -41,29 +42,27 @@ const skin_presets = [
...
 func set_skin(new_skin):
 	for i in range(4):
 		material.set_shader_parameter("color" + str(i + 2), skin_presets[new_skin][i])
 	skin = new_skin
-	temp_skin = new_skin
 
 
 func _ready():
+	set_spot(spot)
+	set_skin(skin)
```
Comment (Koopa1018) [inline classes/interactable/npc/toad/toad.gd:61]: These two function calls aren't necessary--the variables' `set` clauses run them just fine.

**Review by Koopa1018 — APPROVED** [review]
Looking good. Always wondered what these alternate palettes looked like.

File: classes/interactable/npc/toad/toad.gd (hunk 1 above)
Comment (Kuma-Boo) [reply classes/interactable/npc/toad/toad.gd:61]: Those are there so Toads initialize with the correct color when instanced into the editor tree. Without it, you'd need to manually switch the spot and skin properties for the palette to load properly.

Comment (Koopa1018) [reply classes/interactable/npc/toad/toad.gd:61]: Ah. That makes sense. Would suggest mentioning that in a code comment, then (so future readers know not to refactor it out).

**Pull request merged**